### PR TITLE
Add org.kde.keysmith

### PR DESCRIPTION
**Pull request by espidev**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
* dbus is needed for activating the window on an attempt to open a second instance
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://invent.kde.org/utilities/keysmith 
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by espidev** [comment]
bot, build org.kde.keysmith

**Comment by espidev** [comment]
bot, build org.kde.keysmith

File: org.kde.keysmith.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "keysmith",
+    "rename-icon": "keysmith",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--own-name=org.kde.Keysmith"
```
Comment (hfiguiere) [inline org.kde.keysmith.json:13]: Maybe if you set the app-id to match exactly, you don't need to "own-name". That's probably the best.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.keysmith.json (hunk 1 above)
Comment (espidev) [reply org.kde.keysmith.json:13]: Do you mean the "id" field or in the application itself?

Comment (hfiguiere) [reply org.kde.keysmith.json:13]: This probably require to rename the appdata file.

Comment (espidev) [reply org.kde.keysmith.json:13]: org.kde.Keysmith for the "id" seems inconsistent with other kde apps, not sure if there are any implications of that

Comment (hfiguiere) [reply org.kde.keysmith.json:13]: the issue here is that by default the app own the dbus name that match the id, so the two should match. Now the question is whether you are sure of the dbus name?

Comment (nedrichards) [reply org.kde.keysmith.json:13]: Consistency between apps is much less important than matching whatever the app itself expresses - although if we want to be consistent in KDE going into the codebase and changing it isn't *that* hard ;-)

**Comment by espidev** [comment]
bot, build org.kde.keysmith

**Comment by espidev** [comment]
bot, build org.kde.keysmith

**Pull request closed without merging**